src/applications/hca/selectors.js

```diff
@@ -25,8 +25,10 @@ export const isUserLOA3 = state =>
   !hasServerError(state) &&
   !noESRRecordFound(state) &&
   isLOA3(state);
+export const isLoggedOut = state =>
+  !isProfileLoading(state) && !isLoggedIn(state);
 // If we can't get enrollment status for LOA3 users, treat them like a
 // logged-out user (ie, just let them start a new application)
-export const isLoggedOut = state =>
+export const shouldShowLoggedOutContent = state =>
```

Comment:
I renamed the selector from `isLoggedOut` to the more accurate and descriptive `shouldShowLoggedOutContent`. The display logic has gotten more complex because we now have an element that we want to show if the user is actually logged out and only logged out, regardless of if they are in ESR or if there's an ESR server error.